PySpice/Spice/HSpice/hspicefile/hspice_read.c

```
@@ -0,0 +1,1082 @@
+            }
+            if (lastVal > 9e29) {
+                tableBytes -= 8;
+            }
+        }
+    } else {
+        if (tableBytes >= 4) {
+            tableBytes -= 4;
+        }
+    }
+
+    if (isModernFormatGlobal) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)